src/main/java/pl/betoncraft/betonquest/objectives/EntityInteractObjective.java

```diff
@@ -0,0 +1,248 @@
+ * require the player to interact with specifically named mobs and notify them
+ * about the required amount. It can be specified if the player has to
+ * rightclick or damage the entity. Each entity can only be interacted once.
+ * 
+ * @author Jonas Blocher
+ */
+public class EntityInteractObjective extends Objective {
+
+	protected EntityType mobType;
+	protected int amount;
+	protected String name;
+	protected String marked;
+	protected boolean notify;
+	protected Interaction interaction;
+
+	private RightClickListener rightClickListener;
+	private DamageListener damageListener;
+
+	public enum Interaction {
+		RIGHT_CLICK, DAMAGE, ANY
+	}
+
+	public EntityInteractObjective(Instruction instruction) throws InstructionParseException {
+		super(instruction);
+		template = EntityInteractData.class;
+		interaction = instruction.getEnum(Interaction.class);
+		mobType = instruction.getEnum(EntityType.class);
+		amount = instruction.getPositive();
+		name = instruction.getOptional("name");
+		if (name != null) {
+			name = name.replace('_', ' ');
+		}
+		marked = instruction.getOptional("marked");
+		if (marked != null) {
+			marked = Utils.addPackage(instruction.getPackage(), marked);
+		}
+		notify = instruction.hasArgument("notify");
+	}
+
+	@Override
+	public void start() {
+		switch (interaction) {
+		case RIGHT_CLICK:
+			rightClickListener = new RightClickListener();
+			break;
+		case DAMAGE:
+			damageListener = new DamageListener();
+			break;
+		case ANY:
+			rightClickListener = new RightClickListener();
+			damageListener = new DamageListener();
+			break;
+		}
+	}
+
+	private void onInteract(Player player, Entity entity) {
+		// check if it's the right entity type
+		if (!entity.getType().equals(mobType)) {
+			return;
+		}
+		// if the entity should have a name and it does not match, return
+		if (name != null && (entity.getCustomName() == null || !entity.getCustomName().equals(name))) {
+			return;
+		}
+		// check if the entity is correctly marked
+		if (marked != null) {
+			if (!entity.hasMetadata("betonquest-marked")) {
+				return;
+			}
+			List<MetadataValue> meta = entity.getMetadata("betonquest-marked");
+			for (MetadataValue m : meta) {
+				if (!m.asString().equals(marked)) {
+					return;
+				}
+			}
+		}
+		// check if the player has this objective
+		String playerID = PlayerConverter.getID(player);
+		if (containsPlayer(playerID) && checkConditions(playerID)) {
+			// get data off the player
+			EntityInteractData playerData = (EntityInteractData) dataMap.get(playerID);
+			// check if player already interacted with entity
+			if (playerData.containsEntity(entity))
+				return;
+			// right mob is interacted with, handle data update
+			playerData.subtract();
+			playerData.addEntity(entity);
+			if (playerData.isZero()) {
+				completeObjective(playerID);
+			} else if (notify) {
+				// send a notification
+				Config.sendMessage(playerID, "mobs_to_kill", new String[] { String.valueOf(playerData.getAmount()) });
```

Comment:
Tomorrow I'll implement it. I'll also add a german translation and maybe a french one (haven't spoke french for a long time but for this it should be enought^^)